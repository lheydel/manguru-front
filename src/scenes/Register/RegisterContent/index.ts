import { RegisterContent } from './RegisterContent';
export default RegisterContent;
export * from './RegisterContent';
