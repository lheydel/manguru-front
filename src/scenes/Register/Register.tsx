import React from 'react';
import RegisterContent from './RegisterContent';

/**
 * User registration page with its content and all the general elements (header, footer, ...)
 */
export const Register: React.FC = () => (
  <RegisterContent />
);
