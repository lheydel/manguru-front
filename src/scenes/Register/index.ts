import { Register } from './Register';
export default Register;
export * from './Register';
