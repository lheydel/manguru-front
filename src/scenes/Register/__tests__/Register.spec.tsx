import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import Register from '..';

describe('Register component', () => {

  it('should render itself', () => {
    const wrapper = shallow(<Register />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
