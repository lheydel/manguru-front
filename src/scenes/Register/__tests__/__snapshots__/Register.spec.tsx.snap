// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register component should render itself 1`] = `<Connect(RawRegisterContent) />`;
